Ginger/Ginger/Environments/AppsListPage.xaml.cs

```diff
@@ -108,7 +108,8 @@ private void InitReleaseComboBox()
         private void SetGridView()
         {
             //Set the grid name
-            grdApps.Title = $"'{AppEnvironment.Name}' Environment Applications";
+            string grdEnvName = General.EscapeAccessKey(AppEnvironment.Name);
```

Comment:
Create separate function for to sanitize and set grid title

Reply:
Created a separate function